### ISSUE TITLE
emphasis markers interspersed with ruby II

### ISSUE THREAD

**Issue by duncdrum** [martindholmes/rubyForTEI#10]
since the discussion in the other thread #6 moved to another topic here it goes again. 

Interspersing emphasis markers with ruby (or ruby-like) annotations is the most common form of appearance in historical Chinese documents. Encoders, not primarily working with Japanese documents, should find guidance as to how to deal with this from the Guidelines. Not because they don't know what emphasis markers are, but because they introduce significant encoding headaches when trying to encode them, as overlap opportunities abound. It is also far from self-evident of how to do this with respect to the new `ruby` element. 

Once more this is **not** a rare occasion, corner case or exception, but the norm for historical documents containing ruby from China.  

@747 I would suggest that the Guidelines section, ignores the question if emphasis markers and punctuation characters can be distinguished or not for the purpose of demo markup. I find it impossible to determine, if we project a modern expectation onto the document with respect to treating some marks as punctuation in the absence of any visual distinctions, signaling that historical agents actually perceived this as well. Let's, leave that one to encoders.  

There are two examples already in this repo lets pick one assuming that we include this annotations in our definition of ruby #6

**Comment by knagasaki** [comment]
It seems to me that emphasis markers is a different issue from `ruby`. It must be discussed in the case of not only `ruby` but also `app`, `subst`, and `choice` which have been used also in encoding East Asian document. 

Moreover, ambiguousness of the function of emphasis markers is an issue of descriptive/procedural markup or text-/document-oriented encoding. As several solutions are already prepared in the Guidelines implicitly, the East Asian / Japanese SIG would be an appropriate place to discuss and to show some solutions.

**Comment by knagasaki** [comment]
> It seems to me that emphasis markers is a different issue from `ruby`. It must be discussed in the case of not only `ruby` but also `app`, `subst`, and `choice` which have been used also in encoding East Asian document.
> 
> Moreover, ambiguousness of the function of emphasis markers is an issue of descriptive/procedural markup or text-/document-oriented encoding. As several solutions are already prepared in the Guidelines implicitly, the East Asian / Japanese SIG would be an appropriate place to discuss and to show some solutions.

When I participated in a summer school for digital scholarly edition of medieval manuscripts in Europe, I found that many rules of usage of TEI are share among the medievalists while most of them are not written in the Guidelines. The other communities of TEI seems to be as well, as far as I see in SIG meetings during TEI MM. In the outside of the Guidelines, many rules seem to be accumulated among each community. If you are in a hurry for sharing way of encoding Chinese text, you might be able to get a good solution from people in CBETA or Dharma Drum Institute of Liberal Arts. Otherwise, the SIG for East Asian/Japanese may give a place to do it. The activity of the SIG is very slow, but do for the East Asian / Japanese textual cultures.

**Comment by duncdrum** [comment]
> When I participated in a summer school for digital scholarly edition of medieval manuscripts in Europe, I found that many rules of usage of TEI are share among the medievalists while most of them are not written in the Guidelines. 

This is true, however there are different starting points for the coverage of rules and examples governing Western codices vs East Asian documents in the *Guidelines* to begin with. 

I have sign up to the SIG quite some time ago. Traffic being slow is my impression as well. 

The reason I bring up emphasis markers here, however, is not to ask for general guidance on their encoding, but on the specific headaches that they create when encoding `ruby` which are *interspersed* with emphasis markers. Which is the case in a significant portion of historical documents, if not in the majority of Chinese rubricated documents. Far from a small number of cases, edge case, or special use.

**Comment by knagasaki** [comment]
> > When I participated in a summer school for digital scholarly edition of medieval manuscripts in Europe, I found that many rules of usage of TEI are share among the medievalists while most of them are not written in the Guidelines.
> 
> This is true, however there are different starting points for the coverage of rules and examples governing Western codices vs East Asian documents in the _Guidelines_ to begin with.
> 
> I have sign up to the SIG quite some time ago. Traffic being slow is my impression as well.
> 
> The reason I bring up emphasis markers here, however, is not to ask for general guidance on their encoding, but on the specific headaches that they create when encoding `ruby` which are _interspersed_ with emphasis markers. Which is the case in a significant portion of historical documents, if not in the majority of Chinese rubricated documents. Far from a small number of cases, edge case, or special use.

As the 圏点 is not rare in Chinese document, how has it been treated with `<app>`, `<subst>` and `<choice>`? `<app>` may be rare, but `<subst>` and `<choice>` might be added by `@hand`...?

**Comment by knagasaki** [comment]
> > When I participated in a summer school for digital scholarly edition of medieval manuscripts in Europe, I found that many rules of usage of TEI are share among the medievalists while most of them are not written in the Guidelines.
> 
> This is true, however there are different starting points for the coverage of rules and examples governing Western codices vs East Asian documents in the _Guidelines_ to begin with.

Of course, there are different. Then, we've faced much difficulties but can stand on the shoulders of Giants. If community-based activities would not be enough, the Guidelines would become specific use. Even if 圏点 is general in Chinese document, it is better to talk with other TEIers about it if not yet discussed. 

And then, as I wrote some time, the issue of 圏点 should be solved together with other similar elements. It is not an issue only for `<ruby>`. And in the narrow definition of `<ruby>`, we've not yet found co-occurrence of `ruby` and 圏点. In the ancient document in China, I couldn't find `ruby` in the narrow definition (It doesn't mean that the narrow one only for Japan, because I've heard that it occurs in Chinese textbooks to represent pinyin and I found a case in Latin.). 
In the marginal areas in East Asia, Kaeri-ten, Wokoto-ten and some other grammatical punctuation occur in pre-modern text. It is more complex than the issue of 圏点. So, I understand the importance of treatment of 圏点 and I will do it as far as possible. However, in the context of TEI, it should be solved together with `<app>`, `<subst>`, `<choice>`, and `<note>`. So, if treating 圏点 in the Guidelines, it's necessary to show usage with the similar elements. 

> I have sign up to the SIG quite some time ago. Traffic being slow is my impression as well.

Have you singed up here? (I'm very sorry I've not yet understand the Github system so far. I might overlook your sign up...)
https://github.com/orgs/TEI-EAJ/people

**Comment by knagasaki** [comment]
> > When I participated in a summer school for digital scholarly edition of medieval manuscripts in Europe, I found that many rules of usage of TEI are share among the medievalists while most of them are not written in the Guidelines.
> 
> This is true, however there are different starting points for the coverage of rules and examples governing Western codices vs East Asian documents in the _Guidelines_ to begin with.
> 
> I have sign up to the SIG quite some time ago. Traffic being slow is my impression as well.
> 
> The reason I bring up emphasis markers here, however, is not to ask for general guidance on their encoding, but on the specific headaches that they create when encoding `ruby` which are _interspersed_ with emphasis markers. Which is the case in a significant portion of historical documents, if not in the majority of Chinese rubricated documents. Far from a small number of cases, edge case, or special use.



> since the discussion in the other thread #6 moved to another topic here it goes again.
> 
> Interspersing emphasis markers with ruby (or ruby-like) annotations is the most common form of appearance in historical Chinese documents. Encoders, not primarily working with Japanese documents, should find guidance as to how to deal with this from the Guidelines. Not because they don't know what emphasis markers are, but because they introduce significant encoding headaches when trying to encode them, as overlap opportunities abound. It is also far from self-evident of how to do this with respect to the new `ruby` element.
> 
> Once more this is **not** a rare occasion, corner case or exception, but the norm for historical documents containing ruby from China.
> 
> @747 I would suggest that the Guidelines section, ignores the question if emphasis markers and punctuation characters can be distinguished or not for the purpose of demo markup. I find it impossible to determine, if we project a modern expectation onto the document with respect to treating some marks as punctuation in the absence of any visual distinctions, signaling that historical agents actually perceived this as well. Let's, leave that one to encoders.
> 
> There are two examples already in this repo lets pick one assuming that we include this annotations in our definition of ruby #6

It seems to me that your intention on markup is to encode the Chinese document as what we can see without determination of what it means. It is useful and necessary for encoders who are not responsible for the interpretation of the contents, but it is out of scope of our current proposal of `ruby`. I think it is better to realize what you want under `<sourceDoc>`. If you agree, I would suggest to collaborate to made such kind of markup methods and one or more proposals for improvement of the Guidelines for the next phase of revision of the Guidelines. I think it should take more time to discuss it because it will take some time to do so and  tons of Japanese historical document also have several types of emphasis markers which co-occur with `<app>`, `<subst>`, `<choice>` and `<note>`.

**Comment by duncdrum** [comment]
No I signed up via the email link published on `tei-c.org` as did some of my colleagues (this was broken for quite some time). On github organizations are by invitation, if you invite me I would be happy to continue the discussion there.

**Comment by knagasaki** [comment]
I tried to make an example of a markup using `<note>` with emphasis markers under `<sourceDoc>`. Given that encoders can not determine the meaning of the markers, I suggest to use `<hi>`, if it will be independently encoded. As the targeted text of `<note>` is indicated as pseudo-standoff, the `<hi>` for the emphasis markers can be embedded easily. The URIs of the image are inserted compliant with IIIF. It is just an attempt. I hope your comment about not only usage of tags but also entire strategy of encoding.

```xml
   <sourceDoc source="https://iiif.lib.harvard.edu/manifests/drs:26478660">
      <surface
         source="https://iiif.lib.harvard.edu/manifests/drs:26478660/canvas/canvas-26478854.json"
         facs="https://ids.lib.harvard.edu/ids/iiif/26478854/full/1800,/0/default.jpg">
         <line>
            <hi style="white-circle-right">而較之可手遂<note place="right"
               targetEnd="#note0001e">原評󠄁令有以自容</note>命𥁞絶其纓</hi>
            <hi style="sesame-right">尋</hi><anchor type="noteEnd" xml:id="note0001e"/>荘王與𣈆<hi
               style="sesame-right">戰</hi>一人 </line>
      </surface>
   </sourceDoc>

```

**Comment by knagasaki** [comment]
> No I signed up via the email link published on `tei-c.org` as did some of my colleagues (this was broken for quite some time). On github organizations are by invitation, if you invite me I would be happy to continue the discussion there.

Please use here to discuss and to create a specified guidelines. https://github.com/TEI-EAJ/Guidelines_for_Chinese
